### PR TITLE
[SDKv2] Add query lookupOriginalAccountAddress

### PR DESCRIPTION
**Pull request by 0xjinn**
### Description

Port over the lookupOriginalAccountAddress query

### Test Plan

pnpm fmt & pnpm test

### CODE REVIEW

**Comment by 0xjinn** [comment]
Current dependencies on/for this PR:
* main
  * **PR #10442** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-core/10442" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/aptos-labs/aptos-core/10442?utm_source=stack-comment).

**Review by davidiw — COMMENTED** [review]
should we add some e2e test for key rotation here?

File: ecosystem/typescript/sdk_v2/src/internal/account.ts [hunk 1]
```diff
@@ -148,6 +149,36 @@ export async function getResource(args: {
...
+  const originalAddr = await getTableItem({
+    aptosConfig,
+    handle,
+    data: {
+      key: Hex.fromHexInput({ hexInput: addressOrAuthKey }).toString(),
+      key_type: "address",
+      value_type: "address",
+    },
+    options: options,
+  });
```
Comment (gregnazario) [inline ecosystem/typescript/sdk_v2/src/internal/account.ts:177]: I wonder if we should migrate to using view functions, so we don't have to deal with standardization of the table address.  That'll help with address migration.

**Review by gregnazario — COMMENTED** [review]
Let's add an E2E test, since this should be able to run without transaction submission at first, then we can add rotation, which will also test it again.

File: ecosystem/typescript/sdk_v2/src/internal/account.ts (hunk 1 above)
Comment (0xjinn) [reply ecosystem/typescript/sdk_v2/src/internal/account.ts:177]: Ahh that is a good call. I think we can migrate this after the sdkv2 alpha release.

**Comment by 0xjinn** [comment]
Closing this PR. Moving to the sdk v2 new repo

**Pull request closed without merging**